db/migrate/20231210061330_add_created_at_to_fts_targets_with_index.rb

```diff
@@ -0,0 +1,34 @@
+# For auto load
+FullTextSearch::Migration
+
+class AddCreatedAtToFtsTargetsWithIndex < ActiveRecord::Migration[6.1]
+  def change
+    return if reverting?
+
+    add_column :fts_targets, :created_at, :timestamp if !column_exists?(:fts_targets, :created_at)
```

Comment:
Why do we need this `if ...`?

Reply:
FIx: [3917d56](https://github.com/clear-code/redmine_full_text_search/pull/118/commits/3917d5622259820d744bb121461472373a525542)
I'm so sorry I used it for debugging. I deleted unnecessary codes at this commit.